test/apps/iodemo/io_demo.cc

```diff
@@ -2138,11 +2138,13 @@ static void adjust_opts(options_t *test_opts) {
         std::cout << "ignoring chunk size parameter, because it is not supported"
                      " with AM API" << std::endl;
         test_opts->chunk_size = test_opts->max_data_size;
-        return;
+    } else {
+        test_opts->chunk_size = std::min(test_opts->chunk_size,
+                                         test_opts->max_data_size);
     }
 
-    test_opts->chunk_size = std::min(test_opts->chunk_size,
-                                     test_opts->max_data_size);
+    IoDemoRandom::srand(test_opts->random_seed);
+    LOG << "random seed: " << test_opts->random_seed;
```

Comment:
i think better move random_shuffle and the rest of the code here to do_client

Reply:
done

Reply:
but also return srand() in there as well.

Reply:
> but also return srand() in there as well.

done